src/index.js

```diff
@@ -107,14 +93,36 @@ const addTodo = () => {
   addToLocalStorage();
   inputField.value = '';
 };
-
 addButton.addEventListener('click', (e) => {
   e.preventDefault();
   addTodo();
   displayTasks();
 });
-
 document.addEventListener('DOMContentLoaded', () => {
   getFromLocalStorage();
   displayTasks();
-});
\ No newline at end of file
+});
+const completeTask = (status, item) => {
+  const itemId = Number(item.parentNode.parentNode.id);
+  todo[itemId].completed = status;
+  addToLocalStorage();
+};
+const completeStatus = () => {
+  document.addEventListener('click', (event) => {
+    if (event.target.id === 'checkbox') {
+      completeTask(event.target.checked, event.target);
+    }
+  });
+};
+const clearCompletedTask = () => {
+  todo = todo.filter((t) => t.completed !== true);
+  addToLocalStorage();
+  updateIndex();
+  displayTasks();
+};
+completeStatus();
+document.addEventListener('click', (e) => {
+  if (e.target.id === 'clear') {
+    clearCompletedTask();
+  }
+});
```

Comment:
- [ ] Please, consider, as it is required in the project, to create a separate `.js` file, including in it all logic for status updating (complete `true` or `false`). Then try to import this `.js` file as a module in your `index.js` file. In general, the app will work correctly, but you will cover the project requirement 👍 
```
Add a new JavaScript file and import it as a module:
it will contain methods related to the status updates (completed: true / false).
```